frontend/app/src/components/dashboard/EditSnapshotDialog.vue

```diff
@@ -151,7 +151,6 @@ const save = async (): Promise<boolean> => {
 
   payload.balancesSnapshot = data.balancesSnapshot.map(item => ({
     ...item,
-    location: '',
```

Comment:
Is the payload different than the data we get from the endpoint?

Reply:
no it's not different. I use the location in other part, but it should not be sent to backend.